### PR TITLE
2510 uac by sms

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

Need to update casev3 schema to add in sms_template and fulfilment_survey_sms_template tables.
# What has changed

- Added in sms_template and fulfilment_survey_sms_template tables.
- Added patch script
# How to test?

- Run patch script and see if it creates tables.
# Links

[Trello](https://trello.com/c/uQcwIQcQ/)
# Screenshots (if appropriate):

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[Prototype: Send a UAC by SMS by API (13)](https://trello.com/c/uQcwIQcQ/2510-prototype-send-a-uac-by-sms-by-api-13)

File: patches/100_create_sms_fulfilment_tables.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+-- ***********************************************************************************************
```
Comment (AdamHawtin) [inline patches/100_create_sms_fulfilment_tables.sql:1]: If we are going to run this as a patch, the patch version needs updating. But we might be okay to just ground zero this change again for the moment.

Comment (ryangrundy7) [reply patches/100_create_sms_fulfilment_tables.sql]: Removed for now

File: groundzero_ddl/casev3.sql [hunk 2]
```diff
@@ -138,6 +145,13 @@
         primary key (pack_code)
     );
 
+    create table sms_template (
+       pack_code varchar(255) not null,
+        notify_id uuid not null,
```
Comment (NickGrantONS) [inline groundzero_ddl/casev3.sql:150]:
```suggestion
        notify_template_id uuid not null,
```

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
Small comment

**Review by NickGrantONS — APPROVED** [review]
ATs passing. Great work

**Review by liamtoozer — APPROVED** [review]

**Pull request merged**